### PR TITLE
Update the ROADMAP

### PR DESCRIPTION
**Pull request by kairoaraujo**
- Added to the MWV the Roles Simplification features
- Removed feature, that doesn't impact the minimum functionalities
  - Add TLS/SSL for Broker and Result Backend communication (Issue #6): It can be achivied by not exposing the Broker and Result Backend communication.
- Moved features that are not compatibility breaker to MVP
  - Old Metadata retention
  - Support to AWS S3 (Storage) and AWS KMS (Key Vault)
  - Token revocation
- Fixed components milestone/versions
  - MWV: v1.0.0bX
  - MVP: v1.0.X

Signed-off-by: Kairo de Araujo <kdearaujo@vmware.com>

### CODE REVIEW

File: ROADMAP.rst [hunk 1]
```diff
@@ -33,25 +33,28 @@ Components Milestones:
...
-- [ ] Add TLS/SSL for Broker and Result Backend communication (`Issue #6 <https://github.com/vmware/repository-service-tuf/issues/6>`_)
+
 - [x] Public online documentation (`Issue #22 <https://github.com/vmware/repository-service-tuf/issues/22>`_)
-- [ ] Old Metadata retention (`Issue #29 <https://github.com/vmware/repository-service-tuf/issues/29>`_)
-- [ ] Option to Disable the API Authentication/Authorization Feature (`Issue #41 <https://github.com/vmware/repository-service-tuf/issues/41>`_)
+- [x] Implement HTTPS for the Rest API (`Issue #6 <https://github.com/vmware/repository-service-tuf/issues/6>`_)
+- [ ] Data load for migrations (`Issue #188 <https://github.com/vmware/repository-service-tuf/issues/188>`_)
+- [ ] Remove the BIN Keys from Ceremony/Bootstrap Process [Roles simplification] (`Issue #28 <https://github.com/vmware/repository-service-tuf/issues/28>`_)
+- [ ] Remove from the bootstrap the online keys [Roles simplification] (`Issue #207 <https://github.com/vmware/repository-service-tuf/issues/207>`_)
+- [ ] Simplify the metadata bootstrap process [Roles simplification] (`Issue #208 <https://github.com/vmware/repository-service-tuf/issues/208>`_)
```
Comment (MVrachev) [inline ROADMAP.rst:47]: Issue https://github.com/vmware/repository-service-tuf/issues/28 is replaced by those three issues I have highlighted above, correct?

**Review by MVrachev — COMMENTED** [review]
>  - Add TLS/SSL for Broker and Result Backend communication (Issue #6): It can be achivied by not exposing the Broker and Result Backend communication.

Don't we want to mention it at least in MVP even if it's achieved easily?

**Comment by kairoaraujo** [comment]
I see it as nice to have. 
Documentation about best practices on how to deploy the RSTUF, including the details not to expose some services, is a must. 
I think a “feature” for MVP would be documentation with best practices and how to migrate existing data to RSTUF.

**Comment by MVrachev** [comment]
> Documentation about best practices on how to deploy the RSTUF, including the details not to expose some services, is a must.
> I think a “feature” for MVP would be documentation with best practices and how to migrate existing data to RSTUF.

I think both of those points deserve to be in the MVP goal and are required, so I suggest we add them.

File: ROADMAP.rst (hunk 1 above)
Comment (kairoaraujo) [reply ROADMAP.rst:47]: I think not.
The three issues (#28, #207, and #208) are features related to the Roles Simplification (feature/roles-simplification branch). We want to have it before refining again and implementing the feature Key Rotation (#23).

**Comment by kairoaraujo** [comment]
> > Documentation about best practices on how to deploy the RSTUF, including the details not to expose some services, is a must.
> > I think a “feature” for MVP would be documentation with best practices and how to migrate existing data to RSTUF.
> 
> I think both of those points deserve to be in the MVP goal and are required, so I suggest we add them.

Added to the MVP the documentation feature.

**Review by MVrachev — APPROVED** [review]

**Review by joshuagl — APPROVED** [review]

**Comment by joshuagl** [comment]
Nice re-balancing of priorities here! 👍

**Pull request merged**